packages/runtime-node/test/node-environments-manager.unit.ts

```diff
@@ -1,15 +1,17 @@
 import chai, { expect } from 'chai';
 import chaiAsPromised from 'chai-as-promised';
-import { join } from 'path';
+import { dirname, join } from 'path';
 import { createDisposables } from '@wixc3/engine-core';
 import { Application } from '@wixc3/engine-scripts';
 import { createBrowserProvider } from '@wixc3/engine-test-kit';
 
 chai.use(chaiAsPromised);
 
-const nodeEnvironmentFixturePath = join(__dirname, 'fixtures', 'node-env');
-const multiNodeEnvironmentFixturePath = join(__dirname, 'fixtures', 'multi-node-env');
-const socketNodeEnvironmentFixturePath = join(__dirname, 'fixtures', 'multi-socket-node-env');
+const fixturesFolder = join(dirname(require.resolve('@wixc3/engine-scripts/package.json')), 'test/fixtures');
```

Comment:
I think these fixtures should move as well, otherwise you are adding an undeclared dependency on engine-scripts in the tests

Reply:
I do agree to some extent, but most of testing done for these fixtures are done actually in scripts.
I could have copy-pasted the fixtures, but I thought that because this is inside the tests, it's not so bad

Reply:
So maybe move and import from scripts? just to not have the circular deps

Reply:
it will be strange, since these fixtures to some more stuff, used by other tests.
Moving them is a bit strange, because the vast majority of the code of the fixtures is only somewhat related to what is being tested by the node-environments-manager.unit.ts

Reply:
as discussed, moving the fixtures to a different workspace, and re-using them